projects/valtimo/components/src/lib/components/documenten-api-metadata-modal/documenten-api-metadata-modal.component.html

```diff
@@ -0,0 +1,139 @@
+  </div>
+  <div role="footer">
+    <div class="buttons">
+      <v-button (clickEvent)="cancel()" [disabled]="obs.disabled" type="secondary">{{
+        'document.cancel' | translate
+      }}</v-button>
+      <v-button
+        (clickEvent)="save()"
+        [disabled]="obs.disabled || !obs.valid"
+        mdiIcon="upload"
+        type="success"
+        >{{ 'document.upload' | translate }}</v-button
+      >
+    </div>
+  </div>
+</v-modal>
+
+<ng-template #metadataForm let-obs="obs">
+  <ng-container *ngIf="showForm$ | async">
+    <v-form (valueChange)="formValueChange($event)">
+      <v-input
+        [defaultValue]="obs.file?.name"
+        [disabled]="obs.disabled"
+        [fullWidth]="true"
+        [margin]="true"
+        [required]="true"
+        [title]="'document.filename' | translate"
+        [tooltip]="'document.filenameTooltip' | translate"
+        name="filename"
+      >
+      </v-input>
+      <v-input
+        [disabled]="obs.disabled"
+        [margin]="true"
+        [required]="true"
+        [title]="'document.inputTitle' | translate"
+        [tooltip]="'document.inputTitleTooltip' | translate"
+        [widthPx]="400"
+        name="title"
+      >
+      </v-input>
+      <v-input
+        [disabled]="obs.disabled"
+        [margin]="true"
+        [required]="true"
+        [title]="'document.author' | translate"
+        [tooltip]="'document.authorTooltip' | translate"
+        [widthPx]="300"
+        name="author"
+      >
+      </v-input>
+      <v-date-picker
+        [defaultDateIsToday]="true"
+        [disabled]="obs.disabled"
+        [margin]="true"
+        [required]="true"
+        [title]="'document.creationDate' | translate"
+        [tooltip]="'document.creationDateTooltip' | translate"
+        [widthPx]="100"
+        name="creationDate"
+      ></v-date-picker>
+      <v-date-picker
+        [defaultDateIsToday]="true"
+        [disabled]="obs.disabled"
+        [margin]="true"
+        [title]="'document.receiptDate' | translate"
+        [tooltip]="'document.receiptDateTooltip' | translate"
+        [widthPx]="100"
+        name="receiptDate"
+      ></v-date-picker>
+      <v-date-picker
+        [defaultDateIsToday]="true"
+        [disabled]="obs.disabled"
+        [margin]="true"
+        [title]="'document.sendDate' | translate"
+        [tooltip]="'document.sendDateTooltip' | translate"
+        [widthPx]="100"
+        name="sendDate"
+      ></v-date-picker>
+      <v-input
+        [disabled]="obs.disabled"
+        [fullWidth]="true"
+        [margin]="true"
+        [title]="'document.inputDescription' | translate"
+        [tooltip]="'document.inputDescriptionTooltip' | translate"
+        name="description"
+      >
+      </v-input>
+      <v-select
+        [disabled]="obs.disabled"
+        [items]="confidentialityLevelItems$ | async"
+        [margin]="true"
+        [title]="'document.confidentialityLevel' | translate"
```

Comment:
The default value can be 'zaakvertrouwwelijk'. Im sure people will love to have a lot of default values in this form

Reply:
[The story ](https://ritense.tpondemand.com/entity/36897-be-fe-fill-in-document-metadata) does not mention these requirements